aten/src/ATen/native/mps/operations/TriangularOps.mm

```diff
@@ -1,20 +1,239 @@
 //  Copyright © 2022 Apple Inc.
```

Comment:
Hmm, legal nit but @malfet this copyright comment is not correct given the CLA, right?

Reply:
All copyright headers are superseded by top level license, unless it's a 3rd party code. But I don't want to fight this fight today...